.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,16 @@
+Thank you for reporting and issue to google-cloud-python!
+
+If you are reporting an issue or requesting a feature, please search the
+existing open and closed issues to see if there is already work being done.
+
+If you can provide us with the following information it will help us identify
+the cause of your issue more quickly.
+
+1. google-cloud-python version
+2. Python version and virtual environment information
```

Comment:
Same here, why not show them the command to get the information you want? `python --version`, `pip freeze`?